### PR TITLE
Use plone.base and other updates

### PR DESCRIPTION
**Pull request by jensens**
- remove dependency on CMFPlone (use plone.base)
- isort/black (pure)
- pyupgrade --py37-plus
- remove traces of six
- bump version

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
I branched away 4.x, so Plone 5.x tests can be ignored.

**Review by mauritsvanrees — APPROVED** [review]
Thanks!

File: plone/app/vocabularies/__init__.py [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 from zope.schema.vocabulary import SimpleVocabulary
 
+import urllib
 
-_token_parse_py3 = getattr(urllib, 'parse', None)
-_token_parse_py27 = lambda token: urllib.unquote_plus(token).decode('utf8')
+
+_token_parse_py3 = getattr(urllib, "parse", None)
+_token_parse_py27 = lambda token: urllib.unquote_plus(token).decode("utf8")
 parse = _token_parse_py3.unquote if _token_parse_py3 else _token_parse_py27
```
Comment (davisagli) [inline plone/app/vocabularies/__init__.py:13]: @jensens This could be simplified for Python 3 only

Comment (jensens) [reply plone/app/vocabularies/__init__.py:13]: Best as new PR then

**Pull request merged**